Photos/src/commonMain/kotlin/io/nacular/doodle/examples/PhotosApp.kt

```diff
@@ -258,10 +260,17 @@ private class FixedAspectPhoto(private var image: Image): View() {
                 old.height != height -> size = Size(height * aspectRation, height)
             }
         }
+        val tf = TextField("Write something here")
+        tf.size = Size(230, 32)
+
+        children += tf
     }
 
     override fun render(canvas: Canvas) {
+        println("children: ${children.size}")
+        children.forEach { it.render(canvas) }
```

Comment:
The framework will automatically render children, so parents never need to call this explicitly. Remove this call and you'll see the TextField remains visible. This code is technically fine, but TextField's "native" behavior needs to do special work during render that seems to cause this. You can try adding a Button or something else to see that this would technically work fine.

```kotlin
        val tf = PushButton("Write something here").apply {
            behavior = simpleButtonRenderer { button, canvas -> canvas.rect(button.bounds.atOrigin, fill = Red.paint) }
        }
```

But again, no need to explicitly render children like this. Removing that line will remove the disappearing issue.